README.md

```diff
@@ -49,6 +49,12 @@ With Docker image:
 1. docker build -t pg_doorman -f Dockerfile .
 2. docker run -p 6432:6432 -v /path/to/pg_doorman.toml:/etc/pg_doorman/pg_doorman.toml --rm -t -i pg_doorman
 
+With nix-generated Docker image:
+
+1. nix build .#dockerImage
+2. docker load -i result
+3. docker run -p 6432:6432 -v /path/to/pg_doorman.toml:/etc/pg_doorman/pg_doorman.toml --rm -t -i pg_doorman:1.8.2-4894a3a
```

Comment:
> 1.8.2-4894a3a

Can I ask you not to specify a specific version in the readme? As practice has shown, this information is often forgotten. Please leave only an example, but without specifying versions.

Reply:
Alright. Fixed!

Also I need to mention that this tag is auto-generated.